app/Jobs/DownloadAsset.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Enums\AssetType;
+use App\Models\WpOrg\Asset;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Storage;
+
+class DownloadAsset implements ShouldQueue
+{
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
```

Comment:
Kill `SerializesModels` with fire.  Not a blocker because we're not actually using it, I just don't want jobs to have Models in their state in any way (they can use them just fine, they just have to re-query them in handle()).  I banned SerializesModels at my workplace, and lo and behold Redis stopped running out of storage...

Reply:
As for the Queueing traits, we're going to need a queue-runner container at some point so we're not running the job synchronously, but I think that can wait for a follow-up PR.